tasks/system_probe.py

```diff
@@ -800,6 +805,9 @@ def go_package_dirs(packages, build_tags):
             .strip()
             .split("\n")
         )
+        # Some packages may not be available on all architectures, ignore them
+        # instead of reporting empty path names
+        target_packages += [dir for dir in dirs if dir]
```

Comment:
Nit: explicit check looks more clear

```suggestion
        target_packages += [dir for dir in dirs if len(dir) > 0]
```